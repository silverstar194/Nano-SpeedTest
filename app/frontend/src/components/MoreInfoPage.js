import React, {Component} from 'react';
import Header from './Header'
import '../styles/HomePage.css';

class MoreInfoPage extends Component {
    render() {
        return (
            <div className="MoreInfoPage">
                <Header/>
                <h1 className="page-header text-center">More Info Page</h1>
            </div>
        );
    }
}

export default MoreInfoPage;