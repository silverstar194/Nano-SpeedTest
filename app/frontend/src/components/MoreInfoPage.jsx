import React, {Component} from 'react';
import Header from './Header';
import '../styles/HomePage.css';

class MoreInfoPage extends Component {
    render() {
        return (
            <div className='MoreInfoPage'>
                <Header/>
                <h1 className='page-header text-center'>More Info Page</h1>
                <h2>We are talking with the Nano Team about what information they would like us to display here</h2>
            </div>
        );
    }
}

export default MoreInfoPage;