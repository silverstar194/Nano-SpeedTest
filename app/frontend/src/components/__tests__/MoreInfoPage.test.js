import React from 'react';
import renderer from 'react-test-renderer';
import MoreInfoPage from '../MoreInfoPage';
import { Provider } from 'react-redux';
import configureStore from 'redux-mock-store';
import { BrowserRouter} from 'react-router-dom';
import { shallow } from 'enzyme';

const mockStore = configureStore();
let store = mockStore({
    navigation: {
        activeTab: 'Info'
    }
});

const props = {
	history: {}
};

it('shallow renders without crashing', () => {
    shallow(<MoreInfoPage store={store} {...props}/>);
});

it(`renders snapshot correctly`, () => {
    const tree = renderer.create(
        <Provider store={store}>
            <BrowserRouter >
                <MoreInfoPage {...props}/>
            </BrowserRouter>
        </Provider>
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
