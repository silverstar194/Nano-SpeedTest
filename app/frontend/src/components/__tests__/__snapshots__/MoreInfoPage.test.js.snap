// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot correctly 1`] = `
<div
  className="MoreInfoPage"
>
  <div
    className="nav nav-tabs"
  >
    <li
      className="nav-item"
    >
      <a
        className="nav-link "
        href="/"
        onClick={[Function]}
      >
         
        Speed Test
         
      </a>
    </li>
    <li
      className="nav-item"
    >
      <a
        className="nav-link "
        href="/Stats"
        onClick={[Function]}
      >
         
        Statistics
         
      </a>
    </li>
    <li
      className="nav-item"
    >
      <a
        className="nav-link active"
        href="/Info"
        onClick={[Function]}
      >
         
        More Info
         
      </a>
    </li>
  </div>
  <h1
    className="page-header text-center"
  >
    More Info Page
  </h1>
  <h2>
    We are talking with the Nano Team about what information they would like us to display here
  </h2>
</div>
`;
